### ISSUE TITLE
ActionMenu pressed but not expanding bug

### ISSUE THREAD

**Issue by sunghyunzz** [Clans/FloatingActionButton#49]
I found a bug while developing the app using FloatingActionButton library. Following images are screenshots of what happened.

![not_pressed](https://cloud.githubusercontent.com/assets/8149378/7701124/fe308f44-fe5e-11e4-9c1b-15104042a815.png)
The above color of image is when my FloatingActionMenu is not pressed.

![pressed](https://cloud.githubusercontent.com/assets/8149378/7701125/fe5682da-fe5e-11e4-947e-e8e12d9da1b4.png)
The above image is when the ripple effect happened in FloatingActionMenu. 

The thing happened is **when I scroll at the very top of the FloatingActionMenu**, the **ripple effect occurs** FloatingActionMenu (and I think it's actually pressed) but **FloatingActionMenu does not appear.** I used FloatingActionMenu with the recycler view. 

To sum up, Floating Action Menu sometimes does not expand when I clicked in a little bit odd way.
